### PR TITLE
Added ESM support

### PR DESCRIPTION
**Pull request by D3ord3NidAm**

### CODE REVIEW

**Comment by D3ord3NidAm** [comment]
I just realized I could have put those into the same file that is already there.

File: vsc-extension-quickstart.md [hunk 1]
```diff
@@ -2,22 +2,23 @@
 
 ## What's in the folder
 
-* This folder contains all of the files necessary for your extension.
-* `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
-* `snippets/snippets.code-snippets` - the file containing all snippets.
+- This folder contains all of the files necessary for your extension.
+- `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
+- `snippets/snippets.code-snippets` - the file containing all snippets.
+- `snippets/snippetsesm.code-snippets` - the file containing all ESM snippets.
```
Comment (SrIzan10) [inline vsc-extension-quickstart.md:8]: I don't really care about this md file, it came with the generator, I'll just delete it.

**Comment by SrIzan10** [comment]
also, please use conventional commits from now on

**Comment by D3ord3NidAm** [comment]
What exactly is a conventional commit?

**Pull request closed without merging**